### PR TITLE
:pencil2: Fix typo in Appveyor badge URL

### PR DESCRIPTION
**Pull request by juliaogris**
Fixes typo in URL for Appveyor badge.
https://img.shields.io/appveyor/ci/anz-bank/sysl.svg?label=Windows%20build%20%40%20Appveyor%20CI

@anz-bank/sysl-developers

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Sysl
 .. image:: https://img.shields.io/appveyor/ci/anz-bank/sysl.svg?label=Windows%20build%20%40%20Appveyor%20CI
    :target: https://ci.appveyor.com/project/anz-bank/sysl
 
-Sysl (pronounced "sizzle") is a system specification language. Using Sysl, you
+Sysl (pronounce "sizzle") is a system specification language. Using Sysl, you
```
Comment (marcelocantos) [inline README.rst]: The original is correct.

Comment (juliaogris) [reply README.rst]: Sorry, just wanted to trigger another build.... will fix again. Aren't you meant to be on holidays?

Comment (marcelocantos) [reply README.rst]: I am on holidays, and doing what I enjoy! 🤓

Comment (juliaogris) [reply README.rst]: Fix is in. I wish I had something a bit more interesting for you to enjoy then....

**Pull request merged**